### PR TITLE
Updated documentation.md

### PR DESCRIPTION
**Pull request by packtpartner**
Added 3 new books

*Make sure that you generate site HTML with `jekyll build`, and include the changes to the HTML in your pull request also. See README.md for more information. Please remove this message.*

### CODE REVIEW

**Comment by srowen** [comment]
Please regenerate the HTML, as the note above tells you to

**Comment by srowen** [comment]
Ping @packtpartner

**Comment by packtpartner** [comment]
@srowen I'm afraid I do not know how to do it. Could you please help me out?

**Comment by shagunsodhani** [comment]
@packtpartner Refer [this section](https://github.com/apache/spark-website#generating-the-website-html) in [README.md](https://github.com/apache/spark-website/blob/asf-site/README.md).

**Comment by srowen** [comment]
Ping @packtpartner to fix or close this

File: documentation.md [hunk 1]
```diff
@@ -165,6 +165,9 @@ Slides, videos and EC2-based exercises from each of these are available online:
   <li><a href="https://www.packtpub.com/big-data-and-business-intelligence/apache-spark-graph-processing">Apache Spark Graph Processing</a>, by Rindra Ramamonjison (Packt Publishing)</li>
   <li><a href="https://www.packtpub.com/big-data-and-business-intelligence/mastering-apache-spark">Mastering Apache Spark</a>, by Mike Frampton (Packt Publishing)</li>
   <li><a href="http://www.apress.com/9781484209653">Big Data Analytics with Spark: A Practitioner's Guide to Using Spark for Large Scale Data Analysis</a>, by Mohammed Guller (Apress)</li>
+  <li><a href="https://www.packtpub.com/big-data-and-business-intelligence/real-time-big-data-analytics">Real-Time Big Data Analytics</a>, by Sumit Gupta and Shilpi (Packt Publishing)</li>
+  <li><a href="https://www.packtpub.com/big-data-and-business-intelligence/apache-spark-machine-learning-blueprints">Apache Spark Machine Learning Blueprints</a>, by Alex Liu (Packt Publishing)</li>
+  <li><a href="https://www.packtpub.com/hardware-and-creative/learning-iot-particle-photon-and-electron">Learning IoT with Particle Photon and Electron</a>, by Rashid Khan, Kajari Ghoshdastidar and Ajith Vasudevan (Packt Publishing)</li>
```
Comment (steveloughran) [inline documentation.md:170]: don't see that entry having anything to do with spark.

**Comment by HyukjinKwon** [comment]
ping @packtpartner

**Pull request closed without merging**